docs/examples.md

```diff
@@ -0,0 +1,114 @@
+
+Below is a listing of the examples in the repository:
+
+## Beginning algorithms
+
+*    [Hello Qubit](https://github.com/quantumlib/Cirq/blob/master/examples/hello_qubit.py)
+Simple first program showing how to create a quantum circuit.
+
+*    [Deutsch Algorithm](https://github.com/quantumlib/Cirq/blob/master/examples/deutsch.py)
+Textbook example of the simplest quantum advantage.
+
+*    [Bernstein-Vazirani](https://github.com/quantumlib/Cirq/blob/master/examples/bernstein_vazirani.py)
+Textbook algorithm determining a global property of a function with surprisingly few calls to it.
+
+*    [Bell Inequality](https://github.com/quantumlib/Cirq/blob/master/examples/bell_inequality.py)
+Demonstration of a Bell inequality which shows impossibility of local hidden variable theories.
+
+*    [BB84](https://github.com/quantumlib/Cirq/blob/master/examples/bb84.py)
+Textbook algorithm for Quantum Key Distribution
+
+*    [Noisy simulation](https://github.com/quantumlib/Cirq/blob/master/examples/noisy_simulation_example.py)
+How to use a noisy simulator to generate results with amplitiude damping.
+
+*    [Line placement](https://github.com/quantumlib/Cirq/blob/master/examples/place_on_bristlecone.py)
+How to find a line of adjacent qubits on a device.
+
+*    [Quantum Teleportation](https://github.com/quantumlib/Cirq/blob/master/examples/quantum_teleportation.py)
+A demonstration of using 2 classical bits to transport a quantum state from one
+qubit to another.
+
+*    [Super dense coding](https://github.com/quantumlib/Cirq/blob/master/examples/superdense_coding.py)
+Transmit 2 classical bits using one quantum bit.
+
+
+# Intermediate Textbook Algorithms
+
+*    [Grover Algorithm](https://github.com/quantumlib/Cirq/blob/master/examples/grover.py)
+Textbook algorithm for finding a single element hidden within a oracle function.
+
+*    [Quantum Fourier Transform](https://github.com/quantumlib/Cirq/blob/master/examples/quantum_fourier_transform.py)
+A demonstration of a 4-qubit quantum fourier transform (QFT).
+
+*    [Basic Arithmetic](https://github.com/quantumlib/Cirq/blob/master/examples/basic_arithmetic.py)
+Algorithms for adding and multiplying numbers as represented by quantum states.
+
+*    [Phase estimation](https://github.com/quantumlib/Cirq/blob/master/examples/phase_estimator.py)
+Textbook algorithm for phase estimation, i.e. for finding an eigenvalue of a unitary operator.
+
+*    [Shor](https://github.com/quantumlib/Cirq/blob/master/examples/shor.py)
+Quantum algorithm for integer factoring.
+
+*    [QAOA](https://github.com/quantumlib/Cirq/blob/master/examples/qaoa.py)
+Demonstration of the quantum approximation optimization algorithm (QAOA) on a
+max-cut problem.
+
+
+## Intermediate NISQ techniques
+
+*    [XEB](https://github.com/quantumlib/Cirq/blob/master/examples/cross_entropy_benchmarking_example.py)
+Shows how to use cross-entropy benchmarking (XEB) to to estimate fidelity.
```

Comment:
Almost all other descriptions are in the form of a noun-phrase, but this one (and the one below) are verb-phrases. Could you reword? Example: "Fidelity estimation using cross-entropy benchmarking (XEB)."